src/components/media_manager/src/video/video_stream_to_file_adapter.cc

```diff
@@ -1,197 +0,0 @@
-/*
```

Comment:
@anosach-luxoft,
why not remove this file "video_stream_to_file_adapter.cc "? if it is ancient file...